helpers/messages/about.py

```diff
@@ -0,0 +1,42 @@
+
+
+import newrelic.agent
+import os
+from dotenv import load_dotenv
+from helpers.messages._core import message__custom__info
+
+
+@newrelic.agent.background_task(name='helpers.messages.about.message__info__about', group='Task')
+def message__info__about() -> message__custom__info:
+    return message__custom__info(f"""
+__**Demo Helper**__
+DemoHelper is a queue system for online practical where students can add themselves to the
+queue using `!add`.
```

Comment:
ideally everything would be using slash commands
since the development on discord.py has been ceased, I can't guarentee that standard commands will still be operational by the end of the year

Reply:
I've not actually tried the bot out with the new slash commands. Is your re-write using the slash commands only? or does it support both? But ye should remove any reference to the original commands in favour of slash commands in the built in docs.